index.d.ts

```diff
@@ -1,4 +1,5 @@
-import { FastifyPlugin } from 'fastify';
+import { FastifyPluginCallback } from 'fastify';
+import { OpenAPI, OpenAPIV2, OpenAPIV3 } from 'openapi-types';
```

Comment:
Because of this openapi-types needs to be a dependency and not dev-dependency.
See #340

Reply:
Actually, it is landed from here #328

Reply:
Bit this pr was landed after #328 if i see that correctly here. So on master index.d.ts has a dependency on `openapi-types`, but it does not list it as dependency.

Reply:
This PR is merging the #328 by fast-forwarding commit. Actually, I do not touch `index.d.ts` by my own.
`4.0.0` indeed is merging two PR. #328 and #333, you can check the commit history in here to see the detail.

Reply:
Let say in this way, even without this PR, #328 will cause the same problem. If the problem is adding `openapi-types` as devDeps. It is added in #328.